src/components/Banner2.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Box, Flex, Center } from '@chakra-ui/react'
+import Router, { useRouter } from 'next/router';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../src/redux/store';
+import { AiOutlineHome } from 'react-icons/ai'
+import { TfiAngleDoubleRight } from 'react-icons/tfi'
+import Link from 'next/link';
+
+
+const Banner2 = () => {
+    const values = useSelector((state: RootState) => state.categories.data);
+    const router = useRouter()
+    const { id } = router.query as { id: string };
+    const found = values.find((item) => item.id === Number(id));
```

Comment:
please also here make it descriptive name, for example `selectedCategory`

Reply:
Done